lib/widgets/bottom_sheets/chat_piece_jointe_bottom_sheet.dart

```diff
@@ -258,3 +255,26 @@ class _PermissionDeniedWarning extends StatelessWidget {
     );
   }
 }
+
+class _PieceJointeListTile extends StatelessWidget {
+  const _PieceJointeListTile({
+    required this.icon,
+    required this.text,
+    required this.onPressed,
+  });
+
+  final IconData icon;
+  final String text;
+  final void Function() onPressed;
+
+  @override
+  Widget build(BuildContext context) {
+    return ListTile(
+      leading: Icon(icon),
+      contentPadding: EdgeInsets.zero,
+      dense: true,
```

Comment:
Je ne connaissais pas, mais en regardant la. doc, je vois ça `It is not recommended to set [dense](https://api.flutter.dev/flutter/material/ListTile/dense.html) to true when [ThemeData.useMaterial3](https://api.flutter.dev/flutter/material/ThemeData/useMaterial3.html) is true.`. Qu'en penses-tu ?

Reply:
J'ai trouvé une autre alternative plus customisable : 
```
 return ListTile(
      leading: Icon(icon),
      contentPadding: EdgeInsets.zero,
      visualDensity: VisualDensity(vertical: -2),
      title: Text(text, style: TextStyles.textBaseBold),
      onTap: onPressed,
    );
    ```